src/ostree/ot-admin-builtin-status.c

```diff
@@ -30,12 +30,15 @@
 
 static gboolean opt_verify;
 static gboolean opt_skip_signatures;
+static gboolean opt_query_booted;
 
 static GOptionEntry options[]
     = { { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status",
           NULL },
         { "skip-signatures", 'S', 0, G_OPTION_ARG_NONE, &opt_skip_signatures,
           "Print the commit verification status", NULL },
+        { "query-booted", 'Q', 0, G_OPTION_ARG_NONE, &opt_query_booted,
+          "Print whether the default deployment is booted or not", NULL },
```

Comment:
Remember to update the man page too.

Reply:
Yeah the man page for this has zero command line options documented, I guess we can change that man/ostree-admin-status.xml

Reply:
Ah yes, we are not great at it.  Thank you for doing this!